### DISCUSSION TITLE
Reunión seguimiento del proyecto

### DISCUSSION THREAD

**Discussion by MikaOT** [natiicruuz/24-25-IdSw1-SDR#8] [General]
## Solicitud de Reunión

Buenos días, Jorge @paucres :

Queríamos reunirnos contigo para mostrarte el trabajo que hemos realizado hasta el momento y recibir tu aprobación. También nos gustaría aprovechar la ocasión para resolver algunas dudas que han surgido.

¿Podrías indicarnos qué días y horarios te vendrían bien para esta reunión?

Quedamos a la espera de tu respuesta.  
¡Gracias!

Saludos,
Cayetano

**Comment by natiicruuz** [comment]
Buen día @paucres , 

Quedamos a la espera de una fecha para esta semana, en todo caso de no poder coordinar una fecha, le invitamos a que revise el [**modelo del dominio**](https://github.com/natiicruuz/24-25-IdSw1-SDR/blob/main/ModeloDelDominio/modeloDelDominio.md) del proyecto para que nos diga si le parece bien o se cambiaría o agregaría algo más.

**Reply by paucres** [reply]
Hola Natalia @natiicruuz,

Esta semana me ha sido imposible encontrar un hueco para reunirnos. La semana próxima tengo disponibilidad el miércoles de 09:00 a 11:00 y el jueves de 10:00 a 12:00.

Espero que encaje con vuestros horarios.

Un saludo,

Jorge

**Reply by natiicruuz** [reply]
Buenos días @paucres,

La mayoría del equipo podría el miércoles de 10:00 a 11:00

Le informaré a Manuel y pronto le informaré el salón.

Saludos,

**Reply by mmasias** [reply]
Aula -1.1 - Miercoles 13, de 10h a 11h
